packages/extension-browser/scripts/import-hints.js

```diff
@@ -0,0 +1,48 @@
+const writeFile = require('fs').writeFile;
+const package = require('../package.json');
+
+const hintModules = Object.keys(package.devDependencies).filter((name) => {
+    return name.startsWith('@hint/hint-');
+});
+
+// Generate hints.import.ts
+
+const hints = `// autogenerated by scripts/import-hints.js
+
+import { IHintConstructor } from 'hint/dist/src/lib/types';
+
+const hints = [
+${
+    hintModules.map((name) => {
+        return `    ...Object.values<IHintConstructor>(require('${name}'))`;
+    }).join(',\n')
+}
+];
+
+export default hints;
+`;
+
+writeFile(`${__dirname}/../src/shared/hints.import.ts`, hints, (err) => {
+    console.error(err);
```

Comment:
If this is going to be part of `build-release` or similar we need to handle the `exitCode` to make sure we not continue the build process if this script failed.

Reply:
What's our preferred way of handling this? Calling `process.exit()`, throwing an exception, or something else?

Reply:
Nevermind, `eslint` tells me to throw an error. 😊